sendcloud/classes/Connector.php

```diff
@@ -726,7 +726,9 @@ private function addCarrierLogo($carrier_id)
     {
         $logo_src = realpath(dirname(__FILE__).'/../views/img/carrier_logo.jpg');
         $logo_dst = _PS_SHIP_IMG_DIR_.'/'.$carrier_id.'.jpg';
-        return copy($logo_src, $logo_dst);
+        if (!file_exists(_PS_SHIP_IMG_DIR_.'/'.$carrier_id.'.jpg')) {
```

Comment:
Can't we just use` $logo_dst` instead of `_PS_SHIP_IMG_DIR_.'/'.$carrier_id.'.jpg'`?

Reply:
✅ Done. Thanks